### PR TITLE
Feature/1668 filter accounts

### PR DESCRIPTION
**Pull request by Danchev1**

### CODE REVIEW

File: src/core/models/Polkadot/PolkadotAddress/PolkadotAddress.ts [hunk 1]
```diff
@@ -1,18 +1,9 @@
...
-  meta: types.maybeNull(PolkadotAddressMeta),
-  type: types.maybe(
-    types.union(
-      types.literal('ed25519'),
-      types.literal('sr25519'),
-      types.literal('ecdsa'),
-      types.literal('ethereum')
-    )
-  )
+  meta: types.optional(types.frozen(), null),
```
Comment (PAKovalev) [inline src/core/models/Polkadot/PolkadotAddress/PolkadotAddress.ts:5]: what is the type?

File: src/core/models/Polkadot/PolkadotAddress/PolkadotAddress.ts [hunk 2]
```diff
@@ -1,18 +1,9 @@
...
+  publicKey: types.optional(types.frozen(), '')
```
Comment (PAKovalev) [inline src/core/models/Polkadot/PolkadotAddress/PolkadotAddress.ts:6]: is public key `string`?

**Review by PAKovalev — CHANGES_REQUESTED** [review]

File: src/core/models/Polkadot/PolkadotAddress/PolkadotAddress.ts (hunk 2 above)
Comment (Danchev1) [reply src/core/models/Polkadot/PolkadotAddress/PolkadotAddress.ts:6]: Uint8Array coveted to string

Comment (PAKovalev) [reply src/core/models/Polkadot/PolkadotAddress/PolkadotAddress.ts:6]: Maybe `publicKey: types.optional(types.string, '')` ?

**Pull request closed without merging**